Voting.ipynb

```diff
@@ -277,8 +249,9 @@
     "    \n",
     "    #Calculates the total pref for each candidate and appends it to output table\n",
     "    for i in candidates:\n",
-    "        pref_1, pref_2, pref_3, pref_4, pref_5, pref_6, pref_7, pref_8, pref_9, pref_10 = tally(ballots, i)\n",
-    "        output.append([i, str(pref_1), str(pref_2), str(pref_3), str(pref_4), str(pref_5), str(pref_6), str(pref_7), str(pref_8), str(pref_9), str(pref_10)])\n",
+    "        pref = tally(ballots, i)\n",
+    "        temp = [i].extend( [str(elem) for elem in pref] )\n",
```

Comment:
For whatever reason the .extend() function was returning a list of type None.

However you can just do "temp = [i] + [str(elem) for elem in pref]" instead so that's an easy fix.